zenoh_router/tests/test_unit.py

```diff
@@ -0,0 +1,202 @@
+from typing import Any, Callable, Dict  # noqa: F401
+
+from datadog_checks.base import AgentCheck  # noqa: F401
+from datadog_checks.base.stubs.aggregator import AggregatorStub  # noqa: F401
```

Comment:
Just noticed this, sorry. 

A more robust approach would be to conditionally import these items if type-checking. [Some docs](https://docs.python.org/3/library/typing.html#typing.TYPE_CHECKING). Lmk if the ask is clear.

Not a blocker.

Reply:
Thanks, but I will leave it as is: 1) it consistent with other code in this repo, 2) seems we should introduce some workaround for python2 and it's not worth the effort.